requirements/dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.6
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
`make requirements` needs to be run with Python 3.6 until all environments are running 3.9.

Reply:
done